docs/cron-workflows.md

```diff
@@ -58,9 +58,9 @@ More detailed documentation for the specific library used is [documented here](h
 
```

Comment:
I can't comment on unchanged lines but the description of `activeDeadlineSeconds` above should be updated as well

Reply:
Also ideally the [Fields Reference](https://argo-workflows.readthedocs.io/en/latest/fields/) should be updated as well; who's source is in `cron_workflow_types.go`. Although that one isn't incorrect at least, it's just different